src/Mod/CMakeLists.txt

```diff
@@ -1,148 +1,150 @@
+if(BUILD_ADDONMGR)
+  add_subdirectory(AddonManager)
+endif(BUILD_ADDONMGR)
 
-if(BUILD_POINTS)
-  add_subdirectory(Points)
-endif(BUILD_POINTS)
+if(BUILD_ARCH)
+  add_subdirectory(Arch)
+endif(BUILD_ARCH)
+
+if(BUILD_ASSEMBLY)
+  add_subdirectory(Assembly)
+endif(BUILD_ASSEMBLY)
+
+if(BUILD_CLOUD)
+    add_subdirectory(Cloud)
+endif(BUILD_CLOUD)
 
 if(BUILD_COMPLETE)
   add_subdirectory(Complete)
 endif(BUILD_COMPLETE)
 
-if(BUILD_TEST)
-  add_subdirectory(Test)
-endif(BUILD_TEST)
+if(BUILD_DRAFT)
+  add_subdirectory(Draft)
+endif(BUILD_DRAFT)
 
-#add_subdirectory(TemplatePyMod)
+if(BUILD_DRAWING)
+  add_subdirectory(Drawing)
+endif(BUILD_DRAWING)
+
+if(BUILD_FEM)
+    add_subdirectory(Fem)
+endif(BUILD_FEM)
+
+if(BUILD_IDF)
+  add_subdirectory(Idf)
+endif(BUILD_IDF)
 
 if(BUILD_IMAGE)
   add_subdirectory(Image)
 endif(BUILD_IMAGE)
 
+if(BUILD_IMPORT)
+  add_subdirectory(Import)
+endif(BUILD_IMPORT)
+
+if(BUILD_INSPECTION)
+  add_subdirectory(Inspection)
+endif(BUILD_INSPECTION)
+
+if(BUILD_JTREADER)
+  add_subdirectory(JtReader)
+endif(BUILD_JTREADER)
+
+if(BUILD_MATERIAL)
+  add_subdirectory(Material)
+endif(BUILD_MATERIAL)
+
 if(BUILD_MESH)
   add_subdirectory(Mesh)
 endif(BUILD_MESH)
 
+if(BUILD_MESH_PART)
+  add_subdirectory(MeshPart)
+endif(BUILD_MESH_PART)
+
+if(BUILD_OPENSCAD)
+  add_subdirectory(OpenSCAD)
+endif(BUILD_OPENSCAD)
+
 if(BUILD_PART)
   add_subdirectory(Part)
 endif(BUILD_PART)
 
-if(BUILD_MATERIAL)
-  add_subdirectory(Material)
-endif(BUILD_MATERIAL)
-
 if(BUILD_PART_DESIGN)
   add_subdirectory(PartDesign)
 endif(BUILD_PART_DESIGN)
 
+if(BUILD_PATH)
+    add_subdirectory(Path)
+endif(BUILD_PATH)
+
+if(BUILD_PLOT)
+  add_subdirectory(Plot)
+endif(BUILD_PLOT)
+
+if(BUILD_POINTS)
+  add_subdirectory(Points)
+endif(BUILD_POINTS)
+
 if(BUILD_RAYTRACING)
   add_subdirectory(Raytracing)
 endif(BUILD_RAYTRACING)
 
-if(BUILD_DRAWING)
-  add_subdirectory(Drawing)
-endif(BUILD_DRAWING)
+if(BUILD_REVERSEENGINEERING)
+  add_subdirectory(ReverseEngineering)
+endif(BUILD_REVERSEENGINEERING)
 
 IF(EIGEN3_FOUND)
```

Comment:
As long as you're making this change, I'd probably invert the conditionals, so that the message only prints out if the user actually requested Robot (and below, Sketcher). No sense saying that the reason it isn't built is Eigen3, if the user didn't even request it.

Reply:
You are right, this makes sense and unifies the import statement even further. I added another commit with this change, which also unifies the capitalization of the `if` statements.

Check if you are fine with it, then I would squash all commits into a single commit PR before merging.